tests/v0_tests/test_tensor_search.py

```diff
@@ -248,3 +248,25 @@ def test_pagination_single_field(self):
 
                     # TODO: re-add this assert when KNN incosistency bug is fixed
                     # assert full_search_results["hits"] == paginated_search_results["hits"]
+
+    def test_reranker_properties(self):
+        self.client.create_index(index_name=self.index_name_1)
+        d1 = {
+            "doc title": "Very heavy, dense metallic lead.",
+            "abc-123": "some text blah",
+            "an_int": 2,
+            "_id": "my-cool-doc"
+        }
+        self.client.index(self.index_name_1).add_documents([d1], auto_refresh=True)
+        try:
+            res = self.client.index(self.index_name_1).search(
+                q="example of metals",
+                reranker="openai/gpt3-summarise",
+                searchable_attributes=["doc title"],
+                reranker_properties={"api_key": "f"}
+            )
+        except MarqoWebError as e:
+            options = [["api", "key"], ["open", "ai"]]
```

Comment:
Maybe we can add a comment here like "# test the format of the error message" for clarification.